en/vespa-cli.html

```diff
@@ -165,6 +165,9 @@ <h3 id="documents">Documents</h3>
 # Export all documents in "doc" schema, in "open" namespace
 $ vespa visit --selection 'doc AND id.namespace == "open"'
 
+# Export a specific document, including synthetic (generated) fields with an ID like id:en:doc::doc-en-7764
+$ vespa visit --selection 'id.specific == "doc-en-7764"' --field-set '[all]'
```

Comment:
I suggest instead matching the entire ID verbatim. Using `id.specific` here will risks matching unrelated docs such as `id:fluffy:kittens:g=meow:doc-en-7764`.
```suggestion
$ vespa visit --selection 'id == "id:en:doc::doc-en-7764"' --field-set '[all]'
```

Reply:
ok